File: c language/question 3.c
// 3. Write a function that takes three sides of a triangle and checks whether the triangle is equilateral, isosceles, or scalene.
#include <stdio.h>
int main(){
    int sidea;
    printf("enter sidea");
    scanf("%d",&sidea);
    int sideb;
    printf("enter sideb");
    scanf("%d",&sideb);
    int sidec;
    printf("enter sidec");
    scanf("%d",&sidec);
    if(sidea==sideb && sideb==sidec && sidec==sidea){
        printf("it is a equilateral triangle");
    }
    else if(sidea==sideb || sideb==sidec || sidea==sidec){
        printf("it is a isosceles triangle");
    }
    else if(sidea!=sideb && sideb!=sidec && sidea!=sidec){
        printf("it is isosceles triangle");
    }
}

File: Java script/question 3.js
// 3. Write a function that takes three sides of a triangle and checks whether the triangle is equilateral, isosceles, or scalene.
function triangle(sidea,sideb,sidec){
    if(sidea==sideb && sideb==sidec && sidec==sidea){
        console.log("it is a equilateral triangle")
    }
    else if(sidea==sideb || sideb==sidec || sidea==sidec){
        console.log("it is a isosceles triangle")
    }
    else if(sidea!=sideb && sideb!=sidec && sidea!=sidec){
        console.log("it is isosceles triangle")
    }
}
triangle(12,5,10)